docs/docs_environment.yml

```diff
@@ -29,3 +29,7 @@ dependencies:
   - sphinx-copybutton
   - sphinx-panels
   - sphinxcontrib-bibtex
+  - nbsphinx
+  - sphinx-gallery
+  - sphinx-copybutton
+  - jupyter
```

Comment:
is this sufficient?

```suggestion
  - notebook
```

Reply:
`sphinx` renders the notebooks, so it needs to be able to run them. If `jupyter` is necessary or not, I am not terribly sure. I'm happy to try `notebook`